.github/workflows/linux_unit_test.yml

```diff
@@ -0,0 +1,54 @@
+name: Unit tests
+
+on:
+  push:
+    branches: ['**']
+  pull_request:
+    branches: ['**']
+
+jobs:
+  build:
+
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest]
+        python-version: [3.8,3.9]
```

Comment:
Support for python 3.8 will end by the end of this year. Should we consider also testing more recent' versions of python in the matrix? At some point do we plan to stop testing for older versions? How do we currently communicate with users which python versions are actually supported by hnn? 

@jasmainak @rythorpe @ntolley 

https://devguide.python.org/versions/

Reply:
Perhaps let's add up to `3.11` until we figure out what's going on with `3.12`?

Also in terms of communication, perhaps we can explicitly state in the install instructions which versions of python we have listed in the unit tests here. Something like "HNN-core has been tested on python versions 3.8 - 3.11. Other versions may work but we do not explicitly test against them".

Reply:
@hollandjg Do you recall how [autora](https://pypi.org/project/autora/) does this? I thought there was a badge with tested python versions at one point, I don't see it anymore. Do they have a methodology for supporting versions?